.gitignore

```diff
@@ -28,3 +28,4 @@ override.tf.json
 
 # Include tfplan files to ignore the plan output of command: terraform plan -out=tfplan
 # example: *tfplan*
+.terraform-cache-dir
```

Comment:
If the dir exists Terraform by default uses it ?

Reply:
nope ;)

Reply:
ah so we still need something else to instruct it to use this dir ? Kinda confused on why commiting the dir then, when you configure a cache dir location won't terraform create the dir anyway ? :thinking: (the gitignore stuff makes sense)